packages/core/src/test/awsService/ec2/sshKeyPair.test.ts

```diff
@@ -38,25 +38,36 @@ describe('SshKeyUtility', async function () {
         sinon.restore()
     })
 
-    describe('generateSshKeys', async function () {
-        it('generates key in target file', async function () {
-            const contents = await vscode.workspace.fs.readFile(vscode.Uri.file(keyPath))
-            assert.notStrictEqual(contents.length, 0)
-        })
-
-        it('generates unique key each time', async function () {
-            const beforeContent = await vscode.workspace.fs.readFile(vscode.Uri.file(keyPath))
-            keyPair = await SshKeyPair.getSshKeyPair(keyPath, 30000)
-            const afterContent = await vscode.workspace.fs.readFile(vscode.Uri.file(keyPath))
-            assert.notStrictEqual(beforeContent, afterContent)
-        })
-
-        it('uses ed25519 algorithm to generate the keys', async function () {
-            const process = new ChildProcess(`ssh-keygen`, ['-vvv', '-l', '-f', keyPath])
-            const result = await process.run()
-            // Check private key header for algorithm name
-            assert.strictEqual(result.stdout.includes('[ED25519 256]'), true)
-        })
+    it('generates key in target file', async function () {
+        const contents = await vscode.workspace.fs.readFile(vscode.Uri.file(keyPath))
```

Comment:
nit: could use our "fs" module since it's already being imported to this file. The more places we use preferred things, the more likely that new contributions will copy "best practices" :)

Reply:
If the function is supported by our "fs" is there any reason not to use it? I could make another PR replacing some the other uses?

Reply:
> If the function is supported by our "fs" is there any reason not to use it? 

No, just old outdated code.

> I could make another PR replacing some the other uses?

That would be fantastic!